app/src/main/java/com/rahobbs/todo/fragments/DatePickerFragment.java

```diff
@@ -0,0 +1,79 @@
+package com.rahobbs.todo.fragments;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.DatePicker;
+
+import com.rahobbs.todo.R;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+/**
+ * Created by rachael on 2/5/16.
+ * Dialog fragment to control date picker
+ */
+public class DatePickerFragment extends DialogFragment {
+
+    public static final String EXTRA_DATE = "com.rahobbs.todo.date";
+    private static final String ARG_DATE = "date";
+    private DatePicker mDatePicker;
+
+    public static DatePickerFragment newInstance(Date date){
+        Bundle args = new Bundle();
+        args.putSerializable(ARG_DATE, date);
+
+        DatePickerFragment fragment = new DatePickerFragment();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+        Date date = (Date)getArguments().getSerializable(ARG_DATE);
+        Calendar calendar = Calendar.getInstance();
```

Comment:
something to be aware of when dealing with time is timezones... a user might make an event when they're in NYC and then another one when they're in SF so you want to make sure you're checking / adjusting for zones!